src/utils/layout-helpers.ts

```diff
@@ -364,9 +375,9 @@ export function getProjectAddonsInfo(root: string) {
     }
 
     if (version === 1) {
-      // log('isEmberAddon', packagePath);
+      const addonName = packagePath.split('/').pop();
```

Comment:
addon name could be namespaced like `@my-company/my-addon`, we have to handle it somehow or skip this case for now (check for `@` at first char)

Reply:
good point